### PR TITLE
Create and publish Docker image

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: .github/workflows/build-and-push-image.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```
Comment (sbfnk) [inline .github/workflows/build-and-push-image.yaml:22]: what’s the rationale here for using a specific committee vs. `v2`?

File: .github/workflows/build-and-push-image.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
```
Comment (sbfnk) [inline .github/workflows/build-and-push-image.yaml:30]: likewise, why are we using a specific commit?

File: .github/workflows/build-and-push-image.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@f2a1d5e99d037542a71f64918e516c093c6f3fc4
```
Comment (sbfnk) [inline .github/workflows/build-and-push-image.yaml:35]: again, why this commit?

File: docker/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,42 @@
+FROM rhub/ubuntu
```
Comment (sbfnk) [inline docker/Dockerfile:1]: do we need rhub here? Would it not perhaps be preferable to use the official ubuntu image? https://hub.docker.com/_/ubuntu

**Review by sbfnk — CHANGES_REQUESTED** [review]
Thanks! Great stuff. Just a few questions.

File: .github/workflows/build-and-push-image.yaml (hunk 1 above)
Comment (Bisaloo) [reply .github/workflows/build-and-push-image.yaml:22]: It's a security feature.

A git tag can be overwritten at any time, which means an attacker could replace the current `v2` by malicious code which would then be used in our workflow.

As long as we assume that hash collisions are unlikely / impossible, this cannot happen when tagging a specific commit.

**Review by sbfnk — CHANGES_REQUESTED** [review]
Awesome. One last request: can you add a comment about the rationale for adding locales and tzdata (needed for R)? ThankS!

**Pull request merged**